nginx-proxy/nginx.tmpl

```diff
@@ -30,17 +30,17 @@
     {{ $corsOrigin := (first (groupByKeys .Containers "Env.CORS_ALLOW_ORIGIN")) }}
 
     location ~* "^/(?<path>..)(?<path2>..)(.+)?$" {
-        root        /data/result_storage/v2;
+        root        /data/result_storage;
         expires     1M;
 
         {{ if or $allowCors $corsOrigin }}
             {{ template "cors" (dict "CorsOrigin" $corsOrigin) }}
         {{ end }}
         {{ if $autoWebp }}
-            try_files $webp_prefix/$path/$path2$uri =404;
+            try_files $webp_prefix/$file_path =404;
```

Comment:
map above which defines `$webp_prefix` should be changed to default to `default` value. Currently it's defaulting to empty string. (around line 4)

Reply:
Thanks, fixed